src/inspect.js

```diff
@@ -2,15 +2,17 @@ import fs from 'fs';
 import path from 'path';
 import semver from 'semver';
 import logger from '@percy/logger';
-import migrations from './migrations';
-import { run } from './utils';
+import { run, ROOT, migration } from './utils.js';
```

Comment:
Small personal pref, consts should come before functions when imported like this